V2ex/dist/metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+  "author": {
+    "name": "Xunco",
+    "id": "611786551504011305"
+  },
+  "service": "v2ex",
+  "description": {
+    "en": "A Community of Creative Workers. Discuss exciting topics such as programming, design, hardware, games, and more.",
+    "nl": "Een gemeenschap van creatieve werkers. Bespreek spannende onderwerpen zoals programmeren, ontwerp, hardware en games.",
+    "zh_cn": "创意工作者的社区。讨论编程、设计、硬件、游戏等令人激动的话题。",
+    "zh_tw": "創意工作者的社區。討論編程、設計、硬件、遊戲等令人激動的話題。"
+  },
+  "url": ["www.v2ex.com", "fast.v2ex.com", "v2ex.com"],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/kfj4PUe.png",
+  "thumbnail": "https://i.imgur.com/eugm5S4.png",
+  "color": "#000000",
```

Comment:
This isn't necessary, but can you find a color that accustomed to the website's theme? Such as a dark blue or dark gray.

Reply:
Thanks, please wait.